tests/cases/conformance/intl/collatorCompareBoundFunction.ts

```diff
@@ -0,0 +1,18 @@
+// @target: es2022
+// @lib: es2022.intl
```

Comment:
`es2022.intl` doesn't reference lib.es5, so Collator isn't actually defined in your test script (check the baselines).

Either change this to
```suggestion
// @target: es5
// @lib: es5
```

or just get rid of the lib option entirely, and use the default.